.github/workflows/test-application.yaml

```diff
@@ -146,7 +146,7 @@ jobs:
             - name: Run php-cs-fixer
               if: ${{ matrix.php-cs-fixer }}
               run: |
-                  composer global require friendsofphp/php-cs-fixer --prefer-dist --no-interaction
+                  composer global require friendsofphp/php-cs-fixer:^2.17 --prefer-dist --no-interaction
```

Comment:
Should we consider upgrading to 3.0?

Reply:
yes it would be nice to do that at some point. but i would do that in a separate PR 🙂